src/poetry/core/json/__init__.py

```diff
@@ -9,7 +9,11 @@
 
 from fastjsonschema.exceptions import JsonSchemaException
 
+from poetry.core.json import schemas
+from poetry.core.utils._compat import importlib_resources
 
+
+# TODO: Remove this?
```

Comment:
We can put this behind a `__getattr__()` and raise a deprecation warning too, though I doubt anyone's using it?

Reply:
it is used in poetry itself